manager/integration/tests/test_migration.py

```diff
@@ -123,7 +123,59 @@ def test_migration_with_unscheduled_replica(clients, volume_name):  # NOQA
     11. Enable the scheduling for the node and wait for rebuilding complete.
     12. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)
+
+    # local_node : write data / migrate target
+    # hosts[0] : migrate initial node
+    # hosts[1] : node to schedule on/off
+    local_node = common.get_self_host_id()
+    hosts = get_hosts_for_migration_test(clients)
+    schedule_node = client.by_id_node(hosts[1])
+
+    set_node_scheduling(client, schedule_node,
+                        allowScheduling=False, retry=True)
+
+    # Step 2,3,4
+    volume = client.create_volume(name=volume_name, size=SIZE,
+                                  numberOfReplicas=3,
+                                  backingImage="",
+                                  accessMode="rwx", migratable=True)
+    volume = common.wait_for_volume_detached(client, volume_name)
+    volume.attach(hostId=local_node)
+    volume = common.wait_for_volume_degraded(client, volume_name)
+
+    data = common.write_volume_random_data(volume)
+    common.check_volume_data(volume, data)
+
+    volume.detach(hostId="")
+    volume = common.wait_for_volume_detached(client, volume_name)
+
+    # Step 6
+    volume.attach(hostId=hosts[0])
+    volume = common.wait_for_volume_degraded(client, volume_name)
+
+    # Step 7
+    volume.attach(hostId=local_node)
+
+    # Step 8
+    volume = common.wait_for_volume_migration_ready(client, volume_name)
```

Comment:
You should verify the volume is still degraded and the number of new migrated replicas should the same as the original replicas.

Reply:
HI @innobead , I update code for checking replica count after migrate, thank you